### DISCUSSION TITLE
Wazuh Agent connection off domain network

### DISCUSSION THREAD

**Discussion by heinola** [Security-Onion-Solutions/securityonion#8547] [Unsupported Versions]
Lets say that I will be enrolling the agents while on the domain with TCP and  authd password.

I also will be using a FQDN entry to resolve internally and externally and I am setting Palo Alto firewall NAT for 1514 TCP to the internal server.

Is it possible or a way to allow any ip like 0.0.0.0/0 to the Wazuh server from the outside?  Or is there another way this can be achieved.

Reason:  We do not have alot of mobile computers that leave the network but if they do and more so get lost or stolen I could Geo IP them to the last general location last time they where started up.  Just a thought

S02 
Wazuh 3.13.1

**Comment by isaacgolding** [comment]
If you are looking for ways to geo track your mobile computers there are many other methods more suited to that task vs using SO.   Microsoft provides said tracking features built right into online microsoft accounts.  And opening up SO to the world from a security point of view isn't going to be a good idea.   I would suggest that if you really need the mobile devices to phone home that you look into a VPN system that allows remote devices into your lan,     There are lots of VPN solutions out there both paid and free.     SoftEther VPN Project is a good free one. [https://www.softether.org](https://www.softether.org/)
